### PR TITLE
Discover VM's IP from Satellite (not from hypervisor)

### PR DESCRIPTION
**Pull request by lpramuk**
Lets finish the changes in automation-tools, avahi discovery is now running on Satellite (was on hypervisor)

Let me run some tests first ....

### CODE REVIEW

**Comment by lpramuk** [comment]
@ldjebran asked about ginger VMs (= br0 is used) and I think more changes are needed: 
```
if bridge == 'br0':
    self.provisioning_server (ginger)
else:
   settings.server.hostname (VLANs)
```

**Comment by lpramuk** [comment]
Refactored based @ldjebran feedback, thx.
Mr.Ed don't jump the gun this time :-)

File: robottelo/vm.py [hunk 1]
```diff
@@ -186,7 +186,8 @@ def create(self):
         result = ssh.command(
             u'for i in {{1..60}}; do ping -c1 {0}.local && exit 0; sleep 1;'
             u' done; exit 1'.format(self._target_image),
-            self.provisioning_server,
+            self.provisioning_server if self.bridge == 'br0'
+            else settings.server.hostname,  # in VLANs ping from SAT
```
Comment (ldjebran) [inline robottelo/vm.py]: Think creating a variable is more readable:
```python
ping_from_host_name =  self.provisioning_server if self.bridge == 'br0' else settings.server.hostname
result = ssh.command(
              u'for i in {{1..60}}; do ping -c1 {0}.local && exit 0; sleep 1;'
              u' done; exit 1'.format(self._target_image),
              ping_from_host_name
)
```

Comment (lpramuk) [reply robottelo/vm.py]: I read quite nice - ternary operator , but yeah C++ has it nicer :)

Used just once - no need to create var

**Comment by ldjebran** [comment]
@rplevka Please review

**Comment by rplevka** [comment]
@ldjebran yup, the pr is okay, we discussed it with @lpramuk over irc.

thanks guys

**Pull request merged**